po/pl.po

```diff
@@ -1264,6 +1297,9 @@ msgstr "Wczytuję..."
 msgid "Log messages"
 msgstr "Log komunikatów"
 
+msgid "Loop loading:"
+msgstr "Ładowanie pętle"
```

Comment:
"pętli" instead of "pętle"

Reply:
OK. Could you suggest changes with the GitHub interface that I could apply them easier?

Reply:
```suggestion
msgstr "Ładowanie pętli"
```

Reply:
I think I've got them all!